cfour/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv, lib, makeWrapper, gfortran, requireFile, writeTextFile, python3, blas-i8, lapack,
+  mpi ? null
+}:
+# Make sure to have a correct version of BLAS.
+assert
+  lib.asserts.assertMsg
+  (blas-i8.isILP64)
+  "A 64 bit integer implementation of BLAS is required.";
+
+# Only works with OpenMPI
+assert
+  lib.asserts.assertMsg
+  (mpi == null || mpi.pname == "openmpi")
```

Comment:
It won't work with MPICH?

Reply:
It won't. The configure script allows to explicitly select LAM, IntelMPI and OpenMPI and nothing else. I guess that IntelMPI and MPICH are quite similar and could work. But I guess it is an unsupported configuration.

Reply:
I wonder why the developers make that distinction. MPI is a very well standardized interface. IntelMPI uses the MPICH code base and is nearly identical. We could drop this assert since OpenMPI is the default anyway and it would allow to do test builds with MPICH.

Reply:
I have removed this assert. Having MPICH builds certainly makes sense.